src/recordings.js

```diff
@@ -11,3 +11,103 @@ export async function fetchFirstRecordingURL(wordform) {
   let results = await fetchRecordings(wordform)
   return results[0]['recording_url']
 }
+
+
+/**
+ * Render a list of speakers for the user. The speakers' names can be clicked and the user can hear the word in a variety of dialects (if that's the proper word!).
+ *
+ * @param wordform {string} – the term being defined
+ * 
+ */
+export function getSpeakerList(wordform) {  
+  // get the value of the wordform from the page
+  wordform = document.getElementById('data:head').value;
+  const BASE_URL = 'http://sapir.artsrn.ualberta.ca/validation/recording/_search/' + `${wordform}`;
+
+  // setting up the JSON request
+  let xhttp = new XMLHttpRequest();
+  xhttp.open('GET', BASE_URL, true);
+
+  // receiving request information from SAPIR
+  xhttp.onload = function() {
+    let returnedData = JSON.parse(this.response);
+    let numberOfSpeakers = returnedData.length;
+    const recordingsList = document.querySelector('.recordings');
+    const recordingsHeading = document.querySelector('.definition__recordings');
+    
+    // we only want to display our list of speakers once!
+    if (recordingsList.childElementCount < numberOfSpeakers) {
+      displaySpeakerList(returnedData);
+    }
+
+    // the function that displays an individual speaker's name
+    function displaySpeakerList(firstJSONData) {
+      recordingsHeading.insertAdjacentHTML('afterbegin', '<h3 class="explainer">Tap the names below to hear the word said by various speakers.</h3>');
```

Comment:
I think I nailed everything, but this is the one that a fresh pair of eyes would be really good for 😅 I've set up the code for it within the HTML but am absolutely blanking on what best practices for rendering it might be!